package.json

```diff
@@ -9,16 +9,16 @@
     "test": "./node_modules/.bin/mocha"
   },
   "dependencies": {
-    "ip": "~0.3.1",
-    "moment": "~2.8.1"
+    "ip": "~1.1.2",
+    "moment": "~2.12.0"
   },
   "devDependencies": {
-    "async": "~0.9.0",
-    "express": "~4.9.5",
-    "lodash": "~2.4.1",
-    "mocha": "~1.21.4",
-    "redis": "~0.12.1",
-    "should": "~4.0.4",
-    "supertest": "~0.13.0"
+    "async": "~2.0.0-rc.2",
```

Comment:
ah, i think i did `"async": "~2.0.0-"`

not sure if it would pick stable version later on.

but 👍 to use latest stable `1.5.2`